src/analyze/analyze.c

```diff
@@ -1023,7 +1023,7 @@ static int analyze_critical_chain(int argc, char *argv[], void *userdata) {
         }
         unit_times_hashmap = h;
 
-        (void) pager_open(arg_no_pager, false);
+        (void) pager_open(arg_no_pager);
```

Comment:
I thought about this too, and decided to keep a `bool` param because this would be quite a bit more verbose in the common case:
```C
    (void) pager_open(arg_no_pager * PAGER_ARG_NO_PAGER);
```
But now that I think about this, `arg_no_pager` can be changed to `PagerOptions arg_pager`.
OK, I'll rework.

Reply:
yes, precisely i think `PagerFlags arg_pager_flags;` should be used in all those tools we ship.